content/docs/rfcs/67/README.md

```diff
@@ -0,0 +1,200 @@
+
+All Status message types are described in [62/STATUS-PAYLOAD](/spec/62).
+Status Clients MUST transform the following object into a `MESSAGE` as described below -
+
+```go
+
+type StatusMessage struct {
+    SymKey[]     []byte // [optional] The symmetric key used to encrypt the message
+    PublicKey    []byte // [optional] The public key to use for asymmetric encryption
+    Sig          string // [optional] The private key used to sign the message
+    PubsubTopic  string // The Pubsub topic to publish the message to     
+    ContentTopic string // The Content topic to publish the message to
+    Payload      []byte // A serialized representation of a Status message to be sent
+    Padding      []byte // Padding that must be applied to the Payload
+    TargetPeer   string // [optional] The target recipient of the message
+    Ephemeral    bool   // If the message is not to be stored, this is set to `true` 
+}
+
+```
+
+1. A user MUST only provide either a Symmetric key OR an Asymmetric keypair to encrypt the message.
+If both are received, the implementation MUST throw an error.
+2. `WakuMessage.Payload` MUST be set to `StatusMessage.Payload` 
+3. `WakuMessage.Key` MUST be set to `StatusMessage.SymKey`
+4. `WakuMessage.Version` MUST be set to `1`
+5. `WakuMessage.Ephemeral` MUST be set to `StatusMessage.Ephemeral`
+6. `WakuMessage.ContentTopic` MUST be set to `StatusMessage.ContentTopic`
+7. `WakuMessage.Timestamp` MUST be set to the current Unix epoch timestamp (in nanosecond precision)
+
+## 2. `STORE`
+
+> Note: This protocol MUST remain optional according to the user's preferences,
+it MAY be enabled on Light clients as well.
+
+Messages received via [11/WAKU2-RELAY](/spec/11), are stored in a database.
+When Waku node running this protocol is service node, 
+it MUST provide the complete list of network messages.
+Status clients COULD request historical messages from this service node.
+
+The messages that have the `WakuMessage.Ephemeral` flag set to true will not be stored.
+
+The Status client SHOULD provide a method to prune the database of older records to save storage.
+
+## 3. `FILTER`
+
+> Note: This protocol SHOULD be enabled on Light clients.
+
+This protocol SHOULD be used to filter messages based on a given criteria, such as the `Content Topic` of a `MESSAGE`.
+This allows a reduction in bandwidth consumption by the Status client.
+
+### Content filtering protocol identifers:
+
+`filter-subscribe`:
+
+    /vac/waku/filter-subscribe/2.0.0-beta1
+
+`filter-push`:
+
+    /vac/waku/filter-push/2.0.0-beta1
+
+Status clients SHOULD apply a filter for all the `Content Topic` they are interested in, 
+such as `Content Topic` derived from -
+1. 1:1 chats with other users, described in [55/STATUS-1TO1-CHAT](/spec/55)
+2. Group chats
+3. Community Channels, described in [56/STATUS-COMMUNITIES](/spec/56)
+
+## 4. `LIGHTPUSH`
+
+The `LIGHTPUSH` protocol MUST be enabled solely on Light clients.
```

Comment:
In fact, light clients use `LIGHTPUSH` to publish messages to the network, but the service-side protocol SHOULD be installed on all `RELAY` Status Clients in order to provide this service to light clients.